Coverage analysis broken for transpiled code.

Coverage analysis seems to be broken for transpiled code. (although never officially communicated that it works yet).

```
[2018-02-14 11:06:22.960] [ERROR] StrykerCli - an error occurred Error: No source map reference found in transpiled file "/home/travis/build/stryker-mutator/stryker-jest-runner/src/configLoaders/DefaultJestConfigLoader.js". Cannot analyse code coverage. Setting `coverageAnalysis: "off"` in your stryker.conf.js will prevent this error, but forces Stryker to run each test for each mutant.
```

See https://travis-ci.org/stryker-mutator/stryker-jest-runner/jobs/341380731
While earlier it passed: https://travis-ci.org/stryker-mutator/stryker-jest-runner/builds/340096774

--- a/packages/stryker/package.json
+++ b/packages/stryker/package.json
@@ -59,7 +59,7 @@
     "esprima": "^2.7.0",
     "glob": "^7.0.3",
     "inquirer": "^3.0.6",
-    "istanbul-lib-instrument": "^1.9.1",
+    "istanbul-lib-instrument": "^1.9.2",
     "lodash": "^4.17.4",
     "log4js": "^1.1.0",
     "mkdirp": "^0.5.1",

--- a/packages/stryker/src/transpiler/CoverageInstrumenterTranspiler.ts
+++ b/packages/stryker/src/transpiler/CoverageInstrumenterTranspiler.ts
@@ -24,7 +24,7 @@ export default class CoverageInstrumenterTranspiler implements Transpiler {
   private log: Logger;
 
   constructor(private settings: TranspilerOptions, private testFramework: TestFramework | null) {
-    this.instrumenter = createInstrumenter({ coverageVariable: this.coverageVariable });
+    this.instrumenter = createInstrumenter({ coverageVariable: this.coverageVariable, preserveComments: true });
     this.log = getLogger(CoverageInstrumenterTranspiler.name);
   }
 

--- a/packages/stryker/test/unit/transpiler/CoverageInstrumenterTranspilerSpec.ts
+++ b/packages/stryker/test/unit/transpiler/CoverageInstrumenterTranspilerSpec.ts
@@ -44,6 +44,16 @@ describe('CoverageInstrumenterTranspiler', () => {
       expect(instrumentedContent).to.contain('function something(){cov_').and.contain('.f[0]++');
     });
 
+    it('should preserve source map comments', async () => { 
+      const input = [
+        textFile({ mutated: true, content: 'function something() {} // # sourceMappingUrl="something.map.js"' }),
+      ];
+      const output = await sut.transpile(input);
+      expect(output.error).null;
+      const instrumentedContent = (output.outputFiles[0] as TextFile).content;
+      expect(instrumentedContent).to.contain('sourceMappingUrl="something.map.js"');
+    });
+
     it('should create a statement map for mutated files', () => {
       const input = [
         textFile({ name: 'something.js', mutated: true, content: 'function something () {}' }),

